src/lib/__tests__/server-utilities.test.js

```diff
@@ -30,3 +35,21 @@ describe('md5Hash', () => {
     expect(fileNameHash).not.toBe(md5Hash(fileName));
   });
 });
+
+describe('compressJPEG', () => {
+  if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
+  fs.closeSync(fs.openSync('./tmp/test.jpg', 'a')); // Create empty file 'test.jpg'
+  const filePath = './tmp/test.jpg';
+
+  it('overwrites uncompressed image with compressed image', () => {
```

Comment:
I'd make this a bit more comprehensive, how do you know that it didn't do anything at all and actually had it overwritten?

I think I'd use a real image, just pick any 1 (not too big image, maybe a 100kb one) and put it permanently in the repo for this test, and then maybe at the start of the test make a copy of it, compress it, and check that the size is smaller than it was before? Or since it's a test check that it's the expected size as I guess the compression should work the same every time right? I guess it's deterministic, yeah?